src/components/LoginForm/styles.js

```diff
@@ -1,9 +1,10 @@
 import { StyleSheet } from 'react-native';
+import {COLOR_PRIMARY, COLOR_GRAY, COLOR_GRAY1 } from '../../config/styles';
```

Comment:
suggestion for COLOR_GRAY and COLOR_GRAY1 to change COLOR_GRAY_PRIMARY and COLOR_GRAY_SECONDARY. Be specific when you declaring global variables like these. It is fine if they are method local or class local. But not a good practice when comes to things like these

Reply:
Sure! I'll remember that!